ToDoGeo/Repository/Protocol/ToDoRepositoryProtocol.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  ToDoRepositoryProtocol.swift
+//  ToDoGeo
+//
+//  Created by SUN on 7/2/24.
+//
+
+import RxSwift
+
+protocol ToDoRepositoryProtocol {
+    /// ToDo 등록
+    /// - Parameter todo: 등록할 ToDo
+    /// - Returns: 등록 결과
+    func addToDo(_ todo: ToDo) -> Observable<Void>
+    
+    /// todo 목록 불러오기
+    /// - Returns: todo 목록
+    func getToDos() -> Observable<[ToDo]>
```

Comment:
```suggestion
    func getList() -> Observable<[ToDo]>
```

`getToDos()`보다는 `getList`가 어떨까 합니다.

Reply:
[24c2cae](https://github.com/f-lab-edu/ToDoGeo/pull/21/commits/24c2caec6d1c16949f5d17db7d58f7fdd31fa774) 수정한 커밋입니다.